client/components/DetailViewContainer.js

```diff
@@ -341,7 +353,47 @@ export default class DetailViewContainer extends Component {
               </Link>
             }
           />
+          { screen_width < 800 && (
```

Comment:
This is an example of my comparisons with window.innerWidth (saved locally as screen_width) for responsiveness.